src/pages/IDOPage/IDOPage.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react'
+import useStyle from './styles'
+import { Grid, Typography, Box, CardMedia, Button, Input } from '@material-ui/core'
+import { OutlinedButton } from '@components/OutlinedButton/OutlinedButton'
+import icons from '@static/icons'
+import { Status } from '@reducers/solanaWallet'
+import { useSelector } from 'react-redux'
+import { status } from '@selectors/solanaWallet'
+
+const IDOPage = () => {
+  const [inputValue, setInputValue] = useState<string>('')
+  const [mockedAmount, setMockedAmount] = useState<string>('2.3512')
+
+  const classes = useStyle()
+  const walletStatus = useSelector(status)
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) =>
+    setInputValue(e.target.value)
+
+  const getMaxAmount = () => setInputValue(mockedAmount)
+
+  const getButtonMessage = () => {
+    if (walletStatus !== Status.Initialized) {
+      return 'Connect a wallet'
+    } else {
+      return 'Deposit'
+    }
+  }
+
+  const depositAmount = (amount: string) => {
+    if (walletStatus !== Status.Initialized) return
+    if (!+amount) return console.log('Enter the amount')
+
+    console.log(`You deposited ${amount} SOL`)
+  }
+
+  return (
+    <Grid container className={classes.container}>
+      <Grid container className={classes.mainWrapper}>
+        <Grid container className={classes.header}>
+          <Typography component='h1'>IDO</Typography>
+        </Grid>
+        <Grid container className={classes.idoWrapper}>
+          <Grid container className={classes.main}>
+            <Typography className={classes.mainHeader}>Deposit your SOL</Typography>
+            <Grid className={classes.tokenComponentTextContainer}>
+              <Typography className={classes.tokenComponentText}>
+                <span style={{ fontWeight: 600 }}>Est.:</span> 56.0278$
+              </Typography>
+              <Typography className={classes.tokenComponentText}>
+                <span style={{ fontWeight: 600 }}>Balance:</span> {mockedAmount} SOL
+              </Typography>
+            </Grid>
+            <Input
```

Comment:
FYI we already have input like the one from mockups inside out project, we use it on swap page